llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -2070,7 +2070,7 @@ class GeneratedRTChecks {
     }
   }
 
-  InstructionCost getCost() {
+  InstructionCost getCost(Loop *OuterLoop) {
```

Comment:
It might be better to store the inner or outer loop directly in the struct; the inner loop is passed in to `Create` and there are 2 places that check for an outer loop already (via `if (auto *PL = LI->getLoopFor(LoopVectorPreHeader))`)